easybuild/easyconfigs/f/futhark/futhark-0.19.5.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'Tarball'
+
+name = 'futhark'
+
+version = '0.19.5'
+
+homepage = 'https://futhark-lang.org/'
+
+description = '''Futhark is a small programming language designed to be compiled to efficient parallel code. It 
```

Comment:
Bit of a nitpick, but we normally use """ instead of ''' for long `description` text.